lib/tsconfig/4.7.x/ng13/tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+  "extends": "../tsconfig.json",
+  "angularCompilerOptions": {
+    "allowEmptyCodegenFiles": false,
+    "annotateForClosureCompiler": false,
+    "annotationsAs": "static fields",
+    "disableExpressionLowering": true,
+    "disableTypeScriptVersionCheck": false,
+    "enableIvy": true,
+    "enableLegacyTemplate": false,
+    "enableResourceInlining": false,
+    "fullTemplateTypeCheck": true,
+    "generateCodeForLibraries": true,
+    "preserveWhitespaces": false,
+    "skipMetadataEmit": false,
+    "strictMetadataEmit": false,
+    "skipTemplateCodegen": false,
+    "strictTemplates": true,
+    "strictInjectionParameters": true
+  },
+  "target": "es2015"
```

Comment:
Is it required to set the target to `es2015` here ?
If so, could you add a description about this in the commit message ?

Reply:
File remove because no support ng13 with ts4.7